### DISCUSSION TITLE
List all databases in PostgreSQL Server

### DISCUSSION THREAD

**Discussion by angel3cu** [mtxr/vscode-sqltools#760] [Ideas]
I noticed that the driver only lists the database the user is connected to. Is it possible to support displaying all the databases in the list? I could potentially help with some code changes.

**Comment by mtxr** [comment]
we need to check if it's feasiable by the driver.

Show the list it's possible, i'm pretty sure, but we need to understand how would we query having multiple databases shown for a connection.

@angel3cu  Have you tried something there? do you have any ideas we could bring to discussion?

**Reply by KateYoak** [reply]
I think it's a nice idea.  Toggle the active database, just like we toggle active connection.

**Comment by gilbertmosr** [comment]
Any update on this feature? I badly need it.

**Reply by gilbertmosr** [reply]
A workaround would be to add a feature to duplicate existing connection for one to change only the database/connection name.  

Can anyone direct on where the JSON for exiting connections being are saved? It would be helpful to copy/paste and modify. I've added new connection in the settings but there are not showing up on the connections list.

**Reply by gjsjohnmurray** [reply]
> A workaround would be to add a feature to duplicate existing connection for one to change only the database/connection name.
> 
> Can anyone direct on where the JSON for exiting connections being are saved? It would be helpful to copy/paste and modify. I've added new connection in the settings but there are not showing up on the connections list.

In either User or Workspace settings. See https://code.visualstudio.com/docs/getstarted/settings for background information about VS Code's multilevel settings architecture.

**Reply by gilbertmosr** [reply]
Yeah got it's working now; Thanks!

**Comment by gilbertmosr** [comment]
Actually, i've noticed that if you specify `mysql` as the database to connect to, you will see the list of all other databases on the server.

**Comment by RickeyWard** [comment]
I would like to see this feature too, many postgres tools support this and its how many of us are used to working.

**Comment by gjsjohnmurray** [comment]
Related to https://github.com/mtxr/vscode-sqltools/issues/1152
